fpage/templates/fpage/login.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+    
+<head>
+	<title>Login</title>
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP" crossorigin="anonymous">
+
+
+	<style>

```

Comment:
Move the CSS into static folder.

Reply:
Ok i will make a commit of that

Reply:
The css is just for the login and register page so its fine if we keep it there for now when we will work on UI we will try to change it IMO

Reply:
@shrey1608 Creating a seperate file from the starting makes sure that the existing CSS gets applied in all the templates and it will also reduce the repeated code. And leaving a improvement of one issue on future issue is not a good idea.

Reply:
In order to seperate CSS, 
- Create a static folder inside` fpage` app. 
- inside static create a css subfolder and inside css folder create a `styles.css` file
- Add `STATICFILES_DIRS = [os.path.join(BASE_DIR, 'static')]` in settings.py
- Add `{% load static %}` tag in every templates. 
- in base directory add `<link rel="stylesheet" href = "{% static 'css/styles.css' %}">` inside head.

If you still face any issue, ping me.

Reply:
> @shrey1608 Creating a seperate file from the starting makes sure that the existing CSS gets applied in all the templates and it will also reduce the repeated code. And leaving a improvement of one issue on future issue is not a good idea.

Ohh okay got it!